docs/program/apis/cloud.md

```diff
@@ -33,9 +33,7 @@ Cloud app API methods are only available in the Python SDK.
 
 To use the Viam cloud app API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate an [`AppClient`](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient).
 See the following example for reference.
-To find the location secret, go to [Viam app](https://app.viam.com/), and go to the [**Code sample**](https://docs.viam.com/manage/fleet/robots/#code-sample) tab of any of the robots in the location.
-Toggle **Include secret** on and copy the `payload`.
-For the URL, use the address of any of the robots in the location (also found on the **Code sample** tab).
+To find your api key and secret, go to [Viam app](https://app.viam.com/), and navigate to one of your robots' **Security** tab the bottom of your location's page.
```

Comment:
Is this correct? I've tested this with an org api key that I generated from the CLI. Is the API key pair that gets generated on a robot's security page different? I assume that might be different? Would this be more correct?

```suggestion
Use the Viam CLI [to generate an api key for your organization](https://docs.viam.com/manage/cli/#create-an-organization-api-key).
```

Reply:
yeah we autogenerate an api key specific to your robot automatically when it gets created (only has perms scoped to that robot). That will be the API key used on the code samples page and the security page. eventually (although this work isnt done yet) you will also be able to navigate to the org settings page to see _all_ your api keys there. The copy about the robot security tab makes sense to me.